wp-content/themes/core/components/blocks/links/css/links.pcss

```diff
@@ -171,7 +169,7 @@
 
 		position: absolute;
 		left: 0;
-		top: 4px;
+		top: 0;
```

Comment:
Curious about this change?

Reply:
The psudo element cheveron bullets were misaligned. This fixes that.